### PR TITLE
(maint) Update more libraries for java 9

### PR DESCRIPTION
**Pull request by pcarlisle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## [1.7.0]
```
Comment (jonathannewman) [inline CHANGELOG.md]: I don't think 1.6.3 has been released, so 1.6.3 and 1.7.0 should be combined?

Comment (pcarlisle) [reply CHANGELOG.md]: oh, did i not actually release that

Comment (pcarlisle) [reply CHANGELOG.md]: ok updated

Comment (pcarlisle) [reply CHANGELOG.md]: honestly this changelog is pointless

Comment (jonathannewman) [reply CHANGELOG.md]: I tend to agree, but the course has been set.

**Comment by jonathannewman** [comment]
👍

**Pull request merged**